### ISSUE TITLE
ZPool creation doesn't work for slower drives.

### ISSUE THREAD

**Issue by dmikalova** [openzfs/zfs#2582]
I got this error:

> the kernel failed to rescan the partition table: 16
> cannot label 'sdc': try using parted(8) and then provide a specific slice: -1

I believe I had [this issue](https://github.com/zfsonlinux/zfs/issues/1646) where ZPool creation fails if it takes longer than 1 second. I'm on an up to date Arch Linux using ZFS version 0.6.3_r0_g07dabd2_3.15.8_1-1. Brand new drives were showing really slow burst rw on the order of ~20kB/s. Maybe it's because the heads were parking for energy saving or maybe because it's an older motherboard and SATA controller.

The suggested workaround is to keep trying until I lucked out with burst writes. That worked for 2 drives, but not 3. I ended up running `dd if=/dev/sdb of=/dev/null` for each drive at the same time, which increased burst writes to ~20Mb/s, and then ran the zpool create command and it worked fine.

**Comment by Bronek** [comment]
I can confirm that this bugfix fixed two issues for me:
- I was occasionally unable to create a pool on WD "red" 4TB disks
- I was occasionally unable to import a pool when using flag -d /dev/disk/by-id

Both problems gone after I've patched my copy of 0.6.3-1.2 with commit d09a99f

**Comment by CDLuminate** [comment]
Still can't create zpool with this script:
```shell
echo probe
partprobe

echo remove part
parted /dev/sdb rm 1
parted /dev/sdb rm 1

echo dd clean
dd if=/dev/zero of=/dev/sdb bs=512 count=1

echo labelclear
zpool labelclear /dev/sdb

echo probe
partprobe
lsblk

echo zpool create
zpool create -f -o ashift=9 -m /pool/pool4 pool4 /dev/sdb

echo probe
partprobe
lsblk
```
I've tried hundreds of times. And every time I try it just end up with
```
the kernel failed to rescan the partition table: 2
cannot label 'sdb': try using parted(8) and then provide a specific slice: -1
```
Running `dd if=/dev/sdX of=/dev/null` on background doesn't work.

My hard drive is
```
Model: ATA WDC WD4000FYYZ-0 (scsi)
Disk /dev/sdb: 4001GB
Sector size (logical/physical): 512B/512B
```
